Gemfile

```diff
@@ -2,53 +2,53 @@ ruby File.read(".ruby-version").strip
 
 source "https://rubygems.org"
 
-gem "rails", "~> 6.0"
+gem "rails", "~> 6.0.3", ">= 6.0.3.2"
```

Comment:
How come we need this extra level of specificity for Rails? If we just had `gem "rails"` we'd end up with the same result of 6.0.3.2 so not sure why this needs to be treated different to every other gem.

Reply:
Hi there. Please see https://github.com/alphagov/service-manual-frontend/pull/749#discussion_r454243642 as why we have taken this approach.